pennylane/tape/qscript.py

```diff
@@ -1120,7 +1120,7 @@ def to_openqasm(self, wires=None, rotations=True, measure_all=True, precision=No
         # pylint: disable=no-member
         just_ops = QuantumScript(operations)
         operations = just_ops.expand(
-            depth=2, stop_at=lambda obj: obj.name in OPENQASM_GATES
+            depth=3, stop_at=lambda obj: obj.name in OPENQASM_GATES
```

Comment:
Should we just increase this to 5 or 10? I think 10 is probably excessive, but I wouldn't be surprised if there are some operations that need more than 3 iterations, and it will stop if it gets to the `OPENQASM_GATES` earlier, so I don't see any downside to increasing this number a bit more. 

@albi3ro , am I missing anything here? I'm not sure why this was originally capped at 2.

Reply:
I'd say its fine to bump it up to 10. I think it was probably capped before because we didn't tend to have as complicated of templates that had to be decomposed multiple times.

Reply:
Great, thanks!

Reply:
Ok, updated to 10!